src/geometry/quaternion.rs

```diff
@@ -836,6 +835,16 @@ impl<N: Real + fmt::Display> fmt::Display for Quaternion<N> {
     }
 }
 
+impl<N: Real + fmt::Debug> fmt::Debug for Quaternion<N> {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(
+            f,
+            "Quaternion {:?} − ({:?}, {:?}, {:?})",
+            self[3], self[0], self[1], self[2]
+        )
+    }
+}
+
```

Comment:
This is unidiomatic; `Debug` impls are normally more or less valid Rust syntax for the displayed object. This change is inconsistent with the rest of the lib and doesn't seem to add much value over the derived impl.

Reply:
I hate how the dq is displayed without it.

Reply:
I agree with @Ralith, this should be removed. We already have a `Display` impl that does exactly that. So all you need to do to have a nicer display is to use the `{}` formater.